components/Feature/SharePlan/index.js

```diff
@@ -45,6 +47,15 @@ const SharePlan = ({ error, plan, customerUrl, onPlanShared }) => {
     else return number;
   };
 
+  const saveNumber = () => {
+    plan.numbers[0] = numberText;
+    setEditNumber(false);
+  };
+
+  const handleNumberChange = e => {
+    setNumberText(e.target.value);
```

Comment:
Do we also want to change the number in the database for the future?

Reply:
My understanding was that the number change is only for current plan ("per shared plan" on the card https://trello.com/c/vdj7EwEO/68-edit-phone-number). Unless I got that wrong?

Reply:
So it would still be per plan. Each plan has an entry in dynamoDB and that entry also stores numbers that it gets from SV. So you'd update numbers for that entry in dynamoDB and when this plan is reloaded the number would stay the same.
If a new plan for the same user is created, it would get the numbers from SV again. 
I think that was the idea

Reply:
Might be wrong tho